client/src/components/ReviewList.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react'
+import ReviewTile from './ReviewTile'
+
+const ReviewList = ({productId}) => {
+  const [reviews, setReviews] = useState([])
+
+  const getReviews = async () => {
+    try {
+      const response = await fetch(`/api/v1/products/${productId}/reviews`)
+      if (!response.ok){
+        const errorMessage = `${response.status} (${response.statusText})`
+        throw new Error(errorMessage)
+      }
+      const body = await response.json()
+      setReviews(body.reviews)
+    } catch (error) {
+      console.error(`Error in fetch: ${error.message}`)
+    }
+  }
```

Comment:
this is excellent work....but we'll want to remove this request altogether. For the getRecipe request on your show page, you can serialize in the reviews and then pass them down to this review list.